CHANGELOG.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
Does this CHANGELOG.json actually do anything?  I was expecting a `publish` entry in `scripts` to investigate, but I don't see one 🤔

Reply:
Nope, it's just me keeping a manual record.  Having it actually do something would be great, but I figured doing things manually would be better than doing nothing at all.